Vizzuality/half-earth-v3
# Feature/gtag migration

## Update Google Analytics library
### Description
This PR updates GA library, migrating from `analytics.js` to `gtag.js` in order to be prepared to accept GA4 set up.
Custom events have been updated to adapt to proper event structure for the new library. 
### Testing instructions
Check firing events with [google tag assistant](https://chrome.google.com/webstore/detail/tag-assistant-legacy-by-g/kejbdjndbnbjgmefkgdddjlbokphdefk?hl=en)
[Event list could be checked here](https://docs.google.com/spreadsheets/d/1tjfrNItfXPi6A_pjZcFaSlo2iZAzb7IjHNTceQ_fjsg/edit#gid=1307940195)
### Feature relevant tickets
https://vizzuality.atlassian.net/browse/HE-35
